ReportPortal.Tests/ReportPortal.E2E.UI.Framework/Pages/LaunchesPage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.ObjectModel;
+using OpenQA.Selenium;
+using ReportPortal.E2E.UI.Framework.Helpers;
+
+namespace ReportPortal.E2E.UI.Framework.Pages
+{
+    [PageUrl("/ui/#{projectName}/launches/all")]
+    public class LaunchesPage : BasePage
+    {
+        internal ReadOnlyCollection<IWebElement> LaunchesList => Driver.FindElements(By.XPath("//div[contains(@class,'grid__grid')]/div[@data-id]"));
```

Comment:
better to use * instead of div to have more stable locator

Reply:
Done